Test/CoreSDK.Test/Shared/Channel/TelemetryBufferTest.cs

```diff
@@ -26,23 +26,28 @@ public void DefaultValueIsAppropriateForProductionEnvironmentAndUnitTests()
         {
             var buffer = new TelemetryBuffer();
             Assert.Equal(500, buffer.Capacity);
+            Assert.Equal(1000000, buffer.MaximumUnsentBacklogSize);
```

Comment:
At around 1KB per telemetry item, 1million means 1GB of memory. Queuing up items upto 1GB memory in the rare situation where SDK cannot dispatch items seams reasonable...